drivers/hmis/app/graphql/mutations/delete_enrollment.rb

```diff
@@ -0,0 +1,30 @@
+module Mutations
+  class DeleteEnrollment < BaseMutation
+    argument :id, ID, required: true
+
+    field :enrollment, Types::HmisSchema::Enrollment, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(id:)
+      errors = []
+      enrollment = Hmis::Hud::Enrollment.viewable_by(current_user).find_by(id: id)
+
+      if enrollment.present?
+        if enrollment.in_progress?
+          enrollment.destroy
+        else
+          errors << InputValidationError.new('Only in-progress enrollments can be deleted')
```

Comment:
This is fine for now, but eventually we're going to need the ability to delete an enrollment.  It'll need to do a bunch of cleanup, so probably best to just refuse to do it for now.

Reply:
@eanders The Pivotal story said to only allow deletion of in progress enrollments, so I was following that spec. Happy to change the logic if needed though.

Reply:
Yep, that makes sense for now.  Just noting that it's good we have a place to update this later.  There will be some restrictions on who can delete an enrollment and complexity in what it means, so out of scope for now.